ext/opcache/jit/zend_jit_helpers.c

```diff
@@ -2017,7 +2022,12 @@ static zval* ZEND_FASTCALL zend_jit_fetch_obj_is_slow_ex(zend_object *zobj)
 	void **cache_slot = CACHE_ADDR(opline->extended_value & ~ZEND_FETCH_OBJ_FLAGS);
 
 	retval = zobj->handlers->read_property(zobj, name, BP_VAR_IS, cache_slot, result);
-	if (retval == result && UNEXPECTED(Z_ISREF_P(retval))) {
+	if (retval != result) {
```

Comment:
I could also check for `&EG(uninitialized_zval)` but that check may be more expensive and we need to compare `retval` and `result` anyway for the `zend_unwrap_reference` below.
On the other hand, the current approach may cause more memory writes.